### PR TITLE
[backend/frontend]: feature(freeTrials): add OAEV trials tile and refactor #1626

### PR DESCRIPTION
**Pull request by carinelebas**
# Context
- refactor registeredPlatformQueries:
  - split db query for one or several platforms
  - make a new query to retrieve available trials, and associated data instead  of computing everything in the frontend
- Display OpenAEV trial tile 

## How to test
- Test there is no regression on the registered services for OpenCTI/OpenAEV
  - registered platforms and trials are well retrieved and displayed based on their state
  - createTrials button and banner state is well updated depending on the trials state
  - redirection on free trials creation
- Test the OpenAEV trial tile is well displayed and works well
- Test the new tile is not displayed if feature flag is not enabled

## Related issues

- Related to #1626

# Checklist

## Quality

- [x] I consider this work complete and ready for review
- [ ] I tested all features and edge cases
- [x] I refactored code where necessary to improve quality
- [x] I made sure my code meets development guidelines
- [x] I performed a self-review of my code

## Testing

- [x] I wrote test cases for the relevant use cases (unit, integration or e2e tests)
- [ ] (Bug fixes only) I added tests that reproduce and verify the fix

### What tests have been made

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [x] Local manual tests

# Additional information
> Any additional context, dependencies, or concerns reviewers should know about.

### CODE REVIEW

File: apps/portal-front/app/redirect/[identifier]/create-free-trial.ts [hunk 1]
```diff
@@ -41,30 +40,30 @@ export const redirectToCreateFreeTrial = async (
...
-        platform.deployment_request?.type ===
-          DeploymentRequestDeploymentTypeEnum.TRIAL &&
-        platform.deployment_request.counts_in_orga_quota === true
-    );
-
-    if (freeTrials.length > 0) {
+    if (deployedTrials.length > 0) {
       const instanceUrl = new URL(
-        `/app/service/${platformIdentifier === PlatformIdentifierEnum.OPENCTI ? 'opencti' : 'openaev'}_registration/${freeTrials[0]?.id}`,
+        `/app/service/${platformIdentifier === PlatformIdentifierEnum.OPENCTI ? 'opencti' : 'openaev'}_registration/${deployedTrials[0]?.service_instance_id}`,
```
[the remark this answers is not in the document]
Comment (carinelebas) [reply apps/portal-front/app/redirect/[identifier]/create-free-trial.ts]: done in [5cd98df](https://github.com/FiligranHQ/xtm-hub/pull/1675/commits/5cd98df1149cc0a7a246a96d9a21ca0127c6dbd5)
